force-app/main/default/classes/AffiliationsSettingsController.cls

```diff
@@ -47,6 +47,77 @@ public with sharing class AffiliationsSettingsController {
         }
     }
 
+    /************************************************************************************
+     * @description Updates Affiliaions mapping custom settings
+     * @param mappingName
+     * @param accRecordType
+     * @param conPrimaryAfflField
+     * @return An AffiliationsSettingsVModel
+     ************************************************************************************/
+    @AuraEnabled
+    public static String updateAffiliationMappings(String mappingName, String accRecordType, String conPrimaryAfflField) {
+        try {
+            AffiliationsSettingsController.locateAffiliationMappingsService()
+                .updateAffiliationMappings(mappingName, accRecordType, conPrimaryAfflField);
+
+            RecordTypeModel accRecTypeModel = AffiliationsSettingsController.locateRecordTypeService()
+                .getRecordTypeModelFromRecTypeDevName('Account', accRecordType);
+            
+            if (accRecTypeModel == null) {
+                return '';
```

Comment:
Please return accRecordType here if it is null as invalid values can still be 'saved' if they are unchanged.